.github/workflows/main.yml

```diff
@@ -124,12 +139,17 @@ jobs:
         run:  |
           cd build
           make pinetime-mcuboot-app
+          echo ""
 
       - name: Create firmware image
         run:  |
           # The generated firmware binary looks like "pinetime-mcuboot-app-0.8.2.bin"
           ls -l build/src/pinetime-mcuboot-app*.bin
-          ${{ runner.temp }}/mcuboot/scripts/imgtool.py create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header build/src/pinetime-mcuboot-app*.bin build/src/pinetime-mcuboot-app-img.bin
+          IMGTOOL_INPUT_ARGUMENT=$( ls build/src | grep "pinetime-mcuboot-app-[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\.bin" )
+          echo ""
+          echo -e "\u001b[32m ${IMGTOOL_INPUT_ARGUMENT} \u001b[0m"
+          echo ""
+          ${{ runner.temp }}/mcuboot/scripts/imgtool.py create --align 4 --version 1.0.0 --header-size 32 --slot-size 475136 --pad-header build/src/${IMGTOOL_INPUT_ARGUMENT} build/src/pinetime-mcuboot-app-img.bin
           ${{ runner.temp }}/mcuboot/scripts/imgtool.py verify build/src/pinetime-mcuboot-app-img.bin
 
       - name: Create DFU package
```

Comment:
this one also could be removed by enabling DFU in the Cmake step

Reply:
Right! It was not the case before, but now, images and DFU files are generated directly by CMake so that you don't need to add custom step after the build. You can have a look at the docker file to see how it's done : 
 - build : https://github.com/JF002/InfiniTime/blob/develop/docker/build.sh
 - post build : https://github.com/JF002/InfiniTime/blob/develop/docker/post_build.sh.in

Previously, the post build script generated images and DFU, but now it just copies the binaries into the output folder.

Basically, all you have to do is add  `-DBUILD_DFU=1` to your cmake command line.